docs/gradle-plugin/migrations/migration-to-0.7.0.md

```diff
@@ -250,28 +250,19 @@ koverReport {
 ### Renaming Coverage Engines to Coverage Tools
 Previously called engines have been renamed tools.
 
-IntelliJ Engine was renamed to Kover Tool.
-
-It is now impossible to specify the version of the Kover Tool, because it is embedded into the plugin.
-
-To use embedded Kover Tool:
-```
-kover {
-    useKoverTool()
-}
-```
+IntelliJ Engine was built in by default.
```

Comment:
It still was renamed though. I think we can leave this section, just without 'To use embedded Kover tool'